java-warmup.iml

```diff
@@ -1,30 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this file to be present, my PR removes it due to .gitignore?

Reply:
No, it is ok to have it removed.